webapp/src/Controller/Team/SubmissionController.php

```diff
@@ -114,6 +114,7 @@ public function viewAction(Request $request, int $submitId): Response
         $showCompile      = $this->config->get('show_compile');
         $showSampleOutput = $this->config->get('show_sample_output');
         $allowDownload    = (bool)$this->config->get('allow_team_submission_download');
+        $showToolateResult= $this->config->get('show_too-late_result');
```

Comment:
Can we add a space before the `=`? (and realign the other stuff then)

Reply:
Okay I'll change it.